src/transformer/decode.ts

```diff
@@ -913,21 +914,27 @@ export class Decoder {
     const contributors = getContributors(this.modelMap)
       .map((c) => this.decode(c) as ContributorNode)
       .filter(Boolean)
-    const authorNotes = getAuthorNotes(this.modelMap)
-      .map((authorNote) => this.decode(authorNote) as AuthorNotesNode)
-      .filter(Boolean)
-    const content: ManuscriptNode[] = [...contributors, ...authorNotes]
+    const content: ManuscriptNode[] = [...contributors]
     return schema.nodes.contributors.createAndFill(
       {},
       content
     ) as ManuscriptNode
   }
 
+  private createAuthorNotesNode() {
+    const elements = getCorrespondings(this.modelMap)
```

Comment:
We should also include other "author note" items here. @mbetamony can you help Marek with this?

Reply:
Of course, @mvidtman, please let me know when you resume working on this and we can go through it.

Reply:
@mnatsheh, so we added the content back to the `author_notes`, but i think @mvidtman  concern was that the ticket was only to show the `corresp` information and not the the entire `author_notes` content (there's another ticket for that - [LEAN-3754](https://jira.atypon.com/browse/LEAN-3745)). Now i think we can hide the other nodes from display but I'm not quiet sure if we should wait for the entire author-notes designs or just go along with only showing the corresp information for now.

Reply:
We should move everything now (and remove the old node), but only display the corresp info here.